### PR TITLE
[Term Entry] UI and UX Design UIUX (Ideation) Crazy Eights #4412

### PR DESCRIPTION
**Pull request by DevGat718**
### Description

solving the crazy 8's ui/ux issue to be published.

### Type of Change

- Adding a new entry

### Checklist

- [ x] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by ishg-153** [comment]
@DevGat718 while I work on reviewing this entry, move the `crazy-eights` folder to this correct path:
`content/uiux/concepts/crazy-eights/crazy-eights.md`

**Comment by DevGat718** [comment]
> @DevGat718 while I work on reviewing this entry, move the `crazy-eights` folder to this correct path: `content/uiux/concepts/crazy-eights/crazy-eights.md`

why is the path always wrong? I set the path by what the entry says when I get assigned to it.

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+Title: 'Crazy Eights in  UI/UX'
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:2]:
```suggestion
Title: 'Crazy Eights'
```

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Description: 'It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:3]:
```suggestion
Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes.'
```

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights In UX**
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:9]:
```suggestion
```
Not necessary to separately add as a title

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Crazy 8's is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:10]:
```suggestion
**Crazy Eight's** is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DevGat718  I have suggested some changes, make those. 

Apart from these changes, your entry seems plagiarized, so I have to ask you to rewrite it. 

Make all changes to this same PR. But until you remove all plagiarism, this PR will be marked invalid. We can move forward with the reviews once you are done making the change.

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 1 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:2]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 2 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:3]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 3 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:9]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 4 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:10]: done

**Comment by DevGat718** [comment]
as for the plagiarism what stand out to you, I feel I put the information in my own words mostly but the meaning / context of the entry is straight to the point. I guess what i'm getting out is what do you want me to change exactly?

**Comment by DevGat718** [comment]
?? @ishg-153

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:9]: This seems to be taken directly from here:
https://designsprintkit.withgoogle.com/methodology/phase3-sketch/crazy-8s

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DevGat718 I have added an instance of direct plagiarism. I suggest you also make sure none of the other content in this entry is directly taken from another source.

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 5 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:9]: nah I didn't get that information from, seems like many sources have the same verbiage on the topic. No worries ill take core of it.

**Comment by DevGat718** [comment]
ok everything should be good now I made the corrections @ishg-153

**Comment by DevGat718** [comment]
lets get this done ! @ishg-153

**Comment by DevGat718** [comment]
help @SSwiniarski

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:9]:
```suggestion
**Crazy Eights** is a Design Sprint method used as a quick sketching exercise that challenges people to sketch eight distinct ideas in eight minutes, for the problem at hand. The goal is to push beyond the preliminary idea, and to generate a wide variety of solutions.
```
We refrain from using any personal pronouns like you, we, .. in the entry.

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Example
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:11]:
```suggestion
## Steps in Crazy Eights 
```

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
+
+#### Example  
+To properly implement the crazy eight method would be to 1st begin with an idea in mind for your design then setting a 8minute timer and folding a piece of paper 3 times so that you have eight boxes you then start the timer begin sketching and spend no more than 1 minute per box. Do this with other members in your team the more the ideas the better since this is just the drafting stage.
+![crazy 8's example #1](<www.github.com/codecademy/docs/media/crazy8's #1.jpg>) ![8minTimer](www.github.com/codecademy/docs/media/8mintimer.jpg) ![crazy 8's example #2](<www.github.com/codecademy/docs/media/crazy8's #2.png>)
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:13]: I think only one of the crazy 8's example image should be enough. Please remove the other image and the `8mintimer.jpg` image. Also remember to remove these images from the media folder.

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.
+
+#### Example  
+To properly implement the crazy eight method would be to 1st begin with an idea in mind for your design then setting a 8minute timer and folding a piece of paper 3 times so that you have eight boxes you then start the timer begin sketching and spend no more than 1 minute per box. Do this with other members in your team the more the ideas the better since this is just the drafting stage.
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:12]:
```suggestion
The Crazy Eights methodology is used in a team setting to gather a multitude of ideas. To implement this method follow the steps below: 
1. Begin with a problem statement to solve for.
2. Take a sheet of paper and divide it into 8 sections.
3. Set a 8 minute timer.
4. Start the timer and begin sketching solutions for the problem at hand. 

> **Note:** Spend no more than 1 minute per idea. 

Once the activity is complete, the ideas generated by the entire team are used to finalize a solution.
```
I have structured this example to be steps for crazy eights as it makes more sense for this entry and removed other grammatical errors.

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:3]:
```suggestion
Description: 'A quick sketching exercise that challenges people to sketch eight distinct ideas in eight minutes.'
```

File: content/uiux/concepts/crazy-eights/crazy-eights.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+CatalogContent:
```
Comment (ishg-153) [inline content/uiux/concepts/crazy-eights/crazy-eights.md:4]: Refer [this uiux entry](https://github.com/Codecademy/docs/blob/main/content/uiux/concepts/design-methodologies/terms/five-elements-of-ux-design/five-elements-of-ux-design.md?plain=1) and add `Subjects` and  `Tags` to this entries metadata.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DevGat718 I apologize for delaying the review for this entry. Your entry did not follow the `content-standards` and `term-entry-template`. For future entries, please make sure you refer these documentations while creating your entry. 

For this entry I have made direct suggestions so you can make changes.

**Comment by DevGat718** [comment]
sorry been busy as hell ill take care of these, this week sometime @ishg-153

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 6 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:9]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 7 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:11]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 8 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:13]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 9 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:12]: done.

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 10 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:3]: done

File: content/uiux/concepts/crazy-eights/crazy-eights.md (hunk 11 above)
Comment (DevGat718) [reply content/uiux/concepts/crazy-eights/crazy-eights.md:4]: done

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DevGat718 The entry itself looks good to go now, But you need to remove the `.vscode` folder you seem to have accidentally added.

**Review by ishg-153 — APPROVED** [review]
@DevGat718 I have made the necessary changes. This PR is ready for next review.

**Review by letsbuildanything — APPROVED** [review]
@DevGat718
Thank you for contributing to Codecademy Docs!
Good to go for the merge! 🚀

**Pull request merged**